### DISCUSSION TITLE
M124.0.6367.218 - Pin and un pin displaying red square

### DISCUSSION THREAD

**Discussion by validMyID** [Alex313031/thorium#764] [Q&A] [answered]
hello, i just want to try "to-get-back" use this great browser, after my multiple profile messed up (sadly that make me mad and i uninstall it)...

after i install the SSE mini on my windows, i try to pin and unpin  shortcut, it display red box on left bottom, cannot read the text.

(sry cannot give screenshot because i just uninstalled it, ( by accident actually)

**Comment by Alex313031** [comment, marked as answer]
@validMyID Well I need to see what you're talking about.

**Reply by validMyID** [reply]
it's ok, i think that was  "red color" bug on new chrome look,
